### DISCUSSION TITLE
6 Hal Dasar Untuk Berkembang Sebagai Engineer - Rahmat Agung Julians

### DISCUSSION THREAD

**Discussion by giscus** [rahmatagungj/public-rahmatagungjulians.tech#16] [Comments]
# 6 Hal Dasar Untuk Berkembang Sebagai Engineer - Rahmat Agung Julians

Ketahui beberapa pola dan alasan agar Anda dapat berkembang sebagai Engineer

https://www.rahmatagungjulians.tech/blog/6-hal-dasar-untuk-berkembang-sebagai-engineer

**Comment by bagussatoto** [comment]
## Sangat Motivasi sekali cerita personal anda, Sukses terus bang Agung 
### You are beyond amazing
